### ISSUE TITLE
wav/midi files and internet sounds are played at hugely 

### ISSUE THREAD

**Issue by Gouty** [tgstation/tgstation#31575]
So the

**Comment by WJohn** [comment]
Yeah, admins can't control how loud the video will play at, nor display whether or not the video's link/title appears in the chat like with midis.

**Comment by JJRcop** [comment]
The reason I don't want admins to be able to control the volume for internet midis, is that if they get it wrong, players will have to fiddle with their volume to hear it properly, which will affect the next music. Say an admin plays an internet midi quiet on purpose, so players turn the music slider up, then play a really loud one at full volume, which doesn't feel great.
I encourage you to pick music/audio that isn't too loud, so that the player's appropriate audio settings will work for them. If a track isn't mixed properly, then that's a bad track to play.

If this continues to be a large issue I will add a volume slider movable by admins, but those are the reasons I haven't already.

**Comment by Sadboysuss** [comment]
We now enforce .ogg standards for all sound files in the codebase and players can control the volume of internet sounds. Admins can also stop an internet sound while it is playing if it is too loud.
